cocos/editor-support/cocostudio/ActionTimeline/CCBoneNode.cpp

```diff
@@ -112,6 +112,10 @@ void BoneNode::removeChild(Node* child, bool cleanup /* = true */)
 void BoneNode::removeFromBoneList(BoneNode* bone)
 {
     _childBones.eraseObject(bone);
+    auto skeletonNode = dynamic_cast<SkeletonNode*>(bone);
+    if (bone != nullptr) // nested skeleton
```

Comment:
Why this check ? It can't ensure bone is nested skeleton
If bone is autorelease object, then eraseObject only reduce its reference by 1, it will only be deleted before the next loop.
If bone is retained by user, then eraseObject won't delete it neither.

So please find another way to detect whether it's a nested skeleton or not